packages/data/src/components/with-select/test/index.js

```diff
@@ -18,7 +18,19 @@ import withDispatch from '../../with-dispatch';
 import { createRegistry } from '../../../registry';
 import { RegistryProvider } from '../../registry-provider';
 
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable @wordpress/wp-global-usage */
 describe( 'withSelect', () => {
+	const initialScriptDebug = globalThis.SCRIPT_DEBUG;
```

Comment:
These 3 test suites with exceptions seem like a reasonable compromise. Would it add value to document why it is needed here?